### PR TITLE
test: add test for log formatter

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

**Review by cristianmtr — APPROVED** [review]

File: jina/logging/formatter.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
 from .profile import used_memory
 from ..helper import colored
 
+if False:
```
Comment (Yongxuanzhang) [inline jina/logging/formatter.py:9]: Just curious how is 'if False' used here

**Review by Yongxuanzhang — APPROVED** [review]
LGTM!

File: jina/logging/formatter.py (hunk 1 above)
Comment (JoanFM) [reply jina/logging/formatter.py:9]: It is used to have `flake` not complaint when the class of the `type hint` is not imported while avoid the extra import. 

You can see is a `trick` used all over the place

Comment (Yongxuanzhang) [reply jina/logging/formatter.py:9]: Thanks!

**Pull request merged**